plugin-hrm-form/src/components/caseMergingBanners/ContactAddedToCaseBanner.tsx

```diff
@@ -39,8 +39,8 @@ type Props = OwnProps & ReturnType<typeof mapStateToProps> & ReturnType<typeof m
 
 const mapStateToProps = (state: RootState, { taskId }: OwnProps) => {
   const contact = selectContactByTaskSid(state, taskId);
-  const connectedCase = selectCaseByTaskSid(state, taskId);
   const caseId = contact?.savedContact?.caseId;
+  const connectedCase = state[namespace].activeContacts?.caseConnectedToContact[contact.savedContact.id];
```

Comment:
```suggestion
  const connectedCase = state[namespace].activeContacts.caseConnectedToContact[contact?.savedContact?.id];
```

actiuveContacts is always created, so we don't need the null guard there, but the current contact state or the saved contact attached to it might not be defined, so they do need null guards